### ISSUE TITLE
HTML output generation does not show up progress while operation is ongoing

### ISSUE THREAD

**Issue by de-jcup** [de-jcup/eclipse-asciidoctor-editor#355]
## Situation
When a user starts preview in external browser and there is a problem

![image](https://user-images.githubusercontent.com/11644753/119119459-80ab4480-ba2b-11eb-8ee2-4adf5a7bd9a7.png)

One problem here is that there is no information about the search/wait  for HTML output to be rendered.
So it looks like "nothing happens".

## Wanted
When pressing the button a job shall be started and visible in progress view.

**Comment by de-jcup** [comment]
Here an example:

```asciidoc
// SPDX-License-Identifier: MIT
In PDS we need to store job data (e.g. zipped source code).

*At least one storage setup must be defined*- otherwise {pds} server will not start!
You can either define a shared volume (normally a NFS) or a S3 storage.

Look at <<section-gen-config-scope-storage,Storage configuration>> for configuration details.
```

- internal preview was disabled per default (in preferences)
- opened example file
- now switch inside editor to internal preview
- the progress preview is showing up, but not updated
- just wait... and after a while ... :  
![image](https://user-images.githubusercontent.com/11644753/119268522-ad11bd00-bbf3-11eb-8a82-56fac9a647c6.png)
-  pressing now F5 in editor does the job:
![image](https://user-images.githubusercontent.com/11644753/119268548-c286e700-bbf3-11eb-955e-763190ad8283.png)
- when pressing now "show in external browser" it does NOT work! This is a test scenario for #356

**Comment by de-jcup** [comment]
Hmm.. 
- works with Eclipse-RCP-2018-12 without any problems .
- works with fresh install of Eclipse 2021-03
- works still not inside my upgraded 2020-12 to 2021-03 eclipse instance

So ... compared preferences:
- found difference:  "Enable auto build for external preview"  was set in problematic instance to "on"

After changing this in my Eclipse-RCP runtime I had the the same problems, so reproducable now.

**Comment by de-jcup** [comment]
Found the problem: The old preview file was always deleted (because to force browser shows new output). But afterwards.. Only when NO auto build was enabled the build was triggered again-  this was a former optimization, but now it prevents the normal rendering process.

**Comment by de-jcup** [comment]
Reopened the issue, https://github.com/de-jcup/eclipse-asciidoctor-editor/commit/411db2bc93fe6df589bc0a1e21137c82f9ca6ca3 did not fix the missing progress operation but #356 (the preview rendering).

**Comment by de-jcup** [comment]
We do not need to do anything here - build operation does always trigger progress info about preview build.
